lib/std/special/docs/main.js

```diff
@@ -719,6 +726,33 @@
                     payloadHtml += token('var', tokenKinds.Keyword, wantHtml);
                 }
                 return payloadHtml;
+            case typeKinds.Frame:
+                var name = '@Frame(';
+                var fnObj = zigAnalysis.fns[typeObj.fn];
+                var declObj = fnObj.decl && zigAnalysis.decls[fnObj.decl];
+                var fn_name = declObj ? declObj.name : typeObj.fn_name;
+                if (wantHtml) {
+                  name += '<span class="tok-fn">';
+                  if (declObj) {
+                    // TODO look up decl package and import path
```

Comment:
I could use some help here; the decl has an `import` key that leads to the struct that contains the decl, but I'm not sure how I'm supposed to find the chain of `pkgNames` and `declNames` for `navLink` from there.